ebr_trackerbot/storage/sqlite.py

```diff
@@ -0,0 +1,141 @@
+"""
+Slack Bot SQLite Storage
```

Comment:
We need an issue tracking the idea to move this to a generic DB connector in the future

Reply:
Captured in #3